### ISSUE TITLE
Verify entity totals data

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#1991]
Before we can go live with the live data for the entity totals charts we need to verify that the numbers are correct. Currently, they seem very off. 

The changes will soon be propagating to dev (and eventually staging) where it will be easier to compare the live data with the static data that's currently in production.

@LindsayYoung are these using the cost of election formulas that @PaulClark2 gave us?

**Comment by LindsayYoung** [comment]
The script Paul gave me did not work in postgres so this is a different script.

There are oustanding quesions in https://github.com/18F/openFEC/issues/1953 

The sql is [here](https://github.com/18F/openFEC/blob/develop/data/sql_updates/large_aggregates.sql)

I would like @PaulClark2 and/or @jwchumley to take a look.

If we can show them the chart, it will be much easier to check.

**Comment by LindsayYoung** [comment]
Would a join with `fec_election_yr` get rid of candidates not participating in the correct cycle, or is there a better way?

In the committee part, here is a reference to `pclark.ref_pty_host_convention` in the sql you gave me, is there a way for us to replicate that?

Party committee's seem really low. I think I am missing something there. I think I need to de-dupe something in the communication filers bucket too.

I was planning on getting rid of the future stuff- I can do that now.

![screen shot 2016-10-21 at 4 40 22 pm](https://cloud.githubusercontent.com/assets/4406333/19652480/eb0e44b6-99de-11e6-9ed1-f1e294ba972f.png)

Most recent SQL is here:
https://github.com/18F/openFEC/blob/53fd5d75b03313f1cea1d59cd0f6c88649769587/data/sql_updates/large_aggregates.sql

**Comment by LindsayYoung** [comment]
OK, I got the IE duplicate problem and future dates going in the last push.

**Comment by PaulClark2** [comment]
Party SQL -- Adjusted receipts: 
coalesce(ttl_op_exp_per,0) should be coalesce(offests_to_op_exp,0) 

This doesn't complete fix the problem. The party amounts are still off. The PG generated amount are too low.

**Comment by PaulClark2** [comment]
Party SQL -- join question

This join seems like it would return too many rows. There's no restriction on ofec_committee_detail_mv's cycle field.

    from fec_vsum_f3x
    left join
        ofec_committee_detail_mv on committee_id = cmte_id
    where
        most_recent_filing_flag like 'Y'
        and ofec_committee_detail_mv.committee_type in ('X', 'Y')
        and ofec_committee_detail_mv.designation <> 'J'

**Comment by LindsayYoung** [comment]
Right now we are dividing by cycle at the end. That cycle number is made with fec_election_yr [here](https://github.com/18F/openFEC/blob/9339fe3e0076562bdd9b86be411926dd6fa7f7b8/data/sql_updates/committee_history.sql#L4). Is there a better way to do that?

**Comment by noahmanger** [comment]
I made a separate issue for creating the communication filers data. So closing this.
